src/app/__snapshots__/App.spec.ts.snap

```diff
@@ -916,32 +916,27 @@ exports[`App.vue fails to renders basic view 1`] = `
                      Join the Kuma community and ask questions: 
                   </p>
                   <ul>
-                    
                     <li>
                       <a
                         href="https://kuma.io/docs/1.7.x/?utm_source=Kuma&utm_medium=Kuma-GUI"
-                        target="_blank"
```

Comment:
Why don’t you keep the blank target. IMHO,  the blank target shall be the default behavior when the href point to a  different domain from the current one.

Reply:
Oh! Good spot! Yeah agree, all these external links should all be target="_blank", let me take a look at that in a sec, thanks! 🙏

Reply:
K these are back in, thanks for that!